File: adjust_logo.js
const fs = require('fs');
const path = require('path');

const cssFiles = fs.readdirSync('.').filter(f => f.endsWith('.css'));

cssFiles.forEach(file => {
    const original = fs.readFileSync(file, 'utf8');
    let content = original;
    
    // Navbar logo
    content = content.replace(/(\.logo-img\s*\{[^}]*?height:\s*)100px/g, '$1140px');
    
    // Mobile and footer
    content = content.replace(/(\.logo-img\s*\{[^}]*?height:\s*)70px/g, '$1100px');
    content = content.replace(/(\.footer-logo\s*\{[^}]*?height:\s*)70px/g, '$1120px');
    content = content.replace(/(\.footer-brand \.footer-logo\s*\{[^}]*?height:\s*)70px/g, '$1120px');
    
    // Loader logo
    content = content.replace(/(\.loader-logo\s*\{[^}]*?width:\s*)250px/g, '$1350px');
    content = content.replace(/(\.loader-logo\s*\{[^}]*?height:\s*)250px/g, '$1350px');

    // Sidebar logo
    content = content.replace(/(\.sidebar-logo\s*\{[^}]*?width:\s*)60px/g, '$190px');
    content = content.replace(/(\.sidebar-logo\s*\{[^}]*?margin-bottom:\s*)10px/g, '$120px');

    // Login logo
    content = content.replace(/(\.login-logo\s*\{[^}]*?width:\s*)120px/g, '$1160px');

    if (content !== original) {
        fs.writeFileSync(file, content, 'utf8');
        console.log('Updated', file);
    }
});

File: adjust_logo.py
import os
import re

css_files = [f for f in os.listdir('.') if f.endswith('.css')]

for file in css_files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
        
    original = content
    
    # 1. Navbar logo img
    # Look for .logo-img { ... height: 100px; ... }
    content = re.sub(r'(\.logo-img\s*\{[^}]*?height:\s*)100px', r'\g<1>140px', content)
    
    # Mobile logo img: .logo-img { ... height: 70px; ... } 
    # and footer logo: .footer-logo { ... height: 70px; ... }
    content = re.sub(r'(\.logo-img\s*\{[^}]*?height:\s*)70px', r'\g<1>100px', content)
    content = re.sub(r'(\.footer-logo\s*\{[^}]*?height:\s*)70px', r'\g<1>120px', content)
    
    # 2. Loader logo
    # .loader-logo { width: 250px; height: 250px; ... }
    content = re.sub(r'(\.loader-logo\s*\{[^}]*?width:\s*)250px', r'\g<1>350px', content)
    content = re.sub(r'(\.loader-logo\s*\{[^}]*?height:\s*)250px', r'\g<1>350px', content)
    
    # 3. Sidebar logo (admin)
    content = re.sub(r'(\.sidebar-logo\s*\{[^}]*?width:\s*)60px', r'\g<1>90px', content)
    content = re.sub(r'(\.sidebar-logo\s*\{[^}]*?margin-bottom:\s*)10px', r'\g<1>20px', content)

    # 4. Login logo (admin)
    content = re.sub(r'(\.login-logo\s*\{[^}]*?width:\s*)120px', r'\g<1>160px', content)
    
    if content != original:
        with open(file, 'w', encoding='utf-8') as f:
            f.write(content)
        print(f"Updated {file}")
